### DISCUSSION TITLE
【whimsical渲染】 无法显示whimsical动态链接的图片

### DISCUSSION THREAD

**Discussion by DevilGrape2** [tangly1024/NotionNext#2025] [Ideas]
在notion页面中插入whimsical 动态链接的图片，但在博客渲染时，这些图片会不显示。

**复现步骤**
在notion页面中插入whimsical 动态链接的图片，查看渲染后的博客页面

**期望的正常结果**
渲染后的博客页面正常显示whimsical 动态链接的图片

**截图**
![notion插入Whimsical](https://img.redred.xyz/notion%E6%8F%92%E5%85%A5Whimsical+.png)
![博客渲染后的界面](https://img.redred.xyz/%E5%8D%9A%E5%AE%A2%E6%B8%B2%E6%9F%93%E5%90%8E.png)

**环境**

- 操作系统: [windows]
- 浏览器 [chrome]
- 版本 [e.g. 22]

**补充说明**
与问题相关的其它说明

**Comment by tangly1024** [comment]
hi, 我没有使用过whimsical

能否提供相关的示例。

**Comment by DevilGrape2** [comment]
给您提供一个[示意图的链接](https://whimsical.com/5PyDyB74xJmNumrnNiVZXK) ,您可以试一下

**Comment by tangly1024** [comment]
目前还不支持 whimsical 格式的网页渲染，这个特性可能在未来的版本中加入。
